marlowe-chain-sync/gen/Language/Marlowe/Runtime/ChainSync/Gen.hs

```diff
@@ -163,7 +163,7 @@ instance Arbitrary PolicyId where
   arbitrary = PolicyId . unScriptHash <$> arbitrary
 
 instance Arbitrary TokenName where
-  arbitrary = TokenName <$> genBytes
+  arbitrary = TokenName . BS.take 32 <$> genBytes
   shrink = genericShrink
 
 instance Arbitrary Datum where
```

Comment:
FYI, the PLT-8148 branch fixes this long-broken `Arbitrary` instance that was able to produce constructors with negative indices. The new Alonzo library now catch this and cause our tests to fail.